cluster-operator/src/test/java/io/strimzi/operator/cluster/model/KafkaClusterTest.java

```diff
@@ -1858,21 +1858,6 @@ public void testReplicationPortNetworkPolicy() {
         assertThat(rules.contains(clusterOperatorPeerNamespaceWithLabels), is(true));
     }
 
-    @Test
-    public void testReplicationPortNetworkPolicyOnOldKubernetes() {
```

Comment:
@scholzj I have removed these tests now as all version of kubernetes in strimzi are supporting network policies, we can also mend these tests to work -so should we keep them them or is it good to remove them?